templates/kernel/lifecycle.html

```diff
@@ -0,0 +1,784 @@
+        <p>Desktop:</p>
+        <pre><code>sudo apt-get install --install-recommends linux-generic-lts-xenial xserver-xorg-core-lts-xenial xserver-xorg-lts-xenial xserver-xorg-video-all-lts-xenial xserver-xorg-input-all-lts-xenial libwayland-egl1-mesa-lts-xenial</code></pre>
+        <p>Server:</p>
+        <pre><code>sudo apt-get install --install-recommends linux-generic-lts-xenial</code></pre>
+        <p>The 14.04.2 and newer point releases ship with an updated kernel and X stack by default. If you have installed with older media, you can use these instructions to install the newer HWE kernel derived from 16.04 (Xenial).</p>
+        <p>If you run a multiarch desktop (for example, i386 and amd64 on amd64, for gaming or Wine), you may find you need a slightly more involved command:</p>
+        <pre><code>sudo apt-get install --install-recommends linux-generic-lts-xenial xserver-xorg-core-lts-xenial xserver-xorg-lts-xenial xserver-xorg-video-all-lts-xenial xserver-xorg-input-all-lts-xenial libwayland-egl1-mesa-lts-xenial libgl1-mesa-glx-lts-xenial libgl1-mesa-glx-lts-xenial:i386 libglapi-mesa-lts-xenial:i386</code></pre>
+      </div>
+
+      <div class="p-tabs__content" id="installation-16-04" role="tabpanel" aria-labelledby="installation-16-04-tab">
+        <h2>Ubuntu 16.04 LTS &mdash; Xenial Xerus</h2>
+        <p>Desktop:</p>
+        <pre><code>sudo apt-get install --install-recommends linux-generic-hwe-16.04 xserver-xorg-hwe-16.04</code></pre>
+        <p>Server:</p>
+        <pre><code>sudo apt-get install --install-recommends linux-generic-hwe-16.04</code></pre>
+        <p>The 16.04.2 and newer point releases ship with an updated kernel and X stack by default for the desktop. Server installations default to the GA kernel and provide the enablement kernel as optional.The 16.04 HWE Stacks follow the Rolling Update Model.</p>
+      </div>
+
+      <div class="p-tabs__content" id="installation-18-04" role="tabpanel" aria-labelledby="installation-18-04-tab">
+        <h2>Ubuntu 18.04 LTS &mdash; Bionic Beaver</h2>
+        <p>Desktop:</p>
+        <pre><code>sudo apt-get install --install-recommends linux-generic-hwe-18.04 xserver-xorg-hwe-18.04</code></pre>
+        <p>Server:</p>
+        <pre><code>sudo apt-get install --install-recommends linux-generic-hwe-18.04</code></pre>
+        <p>The 18.04.2 and newer point releases will ship with an updated kernel and X stack by default for the desktop. Server installations will default to the GA kernel and provide the enablement kernel as optional.</p>
+        <p>The 18.04 HWE Stacks will follow the Rolling Update Model.</p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Check your support status</h2>
+      <p>To determine if your install is still supported, use <code>hwe-support-status</code> as documented on the <a href="/about/release-cycle">Ubuntu release cycle</a>.</p>
```

Comment:
```suggestion
      <p>To determine if your install is still supported, use <code>hwe-support-status</code>:</p>
```

It’s not documented on that page, nor even proposed for it. (And even if it was, we shouldn’t be sending people to a separate page for a single terminal command.)